test/trae/post.spec.ts

```diff
@@ -5,6 +5,8 @@ import nock from 'nock';
 import fetch from 'node-fetch';
 import trae from '../../src';
 
+global.Headers = fetch.Headers;
```

Comment:
`node-fetch` doesn't set the mocks by default

https://github.com/swiftype/swiftype-app-search-javascript/issues/56#issuecomment-468268714